Podcast/SearchTableViewController.swift

```diff
@@ -13,6 +13,7 @@ enum SearchType {
     case episodes
     case series
     case people
+    case itunes
```

Comment:
I get this error when clicking (search Itunes directly)
<img width="619" alt="screen shot 2017-11-01 at 11 35 38 am" src="https://user-images.githubusercontent.com/14966713/32282761-db97ccd8-bef8-11e7-924c-15d04cce0afd.png">
<img width="1117" alt="screen shot 2017-11-01 at 11 35 26 am" src="https://user-images.githubusercontent.com/14966713/32282769-e007d272-bef8-11e7-9e68-9baf7f9bedc6.png">